packages/runtime/runtime-utils/src/summarizerNode.ts

```diff
@@ -599,7 +599,10 @@ export class SummarizerNode implements ISummarizerNode {
         private readonly initialSummary?: IInitialSummary,
     ) {
         this.canReuseHandle = config.canReuseHandle ?? true;
-        this.throwOnError = config.throwOnFailure ?? false;
+        // BUGBUG: Seeing issues with differential summaries.
+        // this will disable them, and throw instead
+        // why we continue to investigate
+        this.throwOnError = config.throwOnFailure ?? true;
```

Comment:
I think we should either force to `true` here or change at all constructor call stacks.

Reply:
+1 to just set it to true unconditionally.